.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,38 @@
+name: 🎉 Feature request
+description: Suggest an idea for the Vault SDK
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting a feature request!
+
+        Please describe what you would like to change/add and why in detail by filling out the template below.
+
+        If you are not sure if your request fits into Dex, you can contact us via the available [support channels](https://github.com/bank-vaults/vault-sdk/issues/new/choose).
+  - type: checkboxes
+    attributes:
+      label: Preflight Checklist
+      description: Please ensure you've completed all of the following.
+      options:
+        - label: I have searched the [issue tracker](https://www.github.com/bank-vaults/vault-sdk/issues) for an issue that matches the one I want to file, without success.
+          required: true
```

```suggestion
```suggestion
          required: true
        - label: I agree to follow the [Code of Conduct](https://bank-vaults.dev/docs/code-of-conduct/).
          required: true
```
```